<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homewindow</class>
 <widget class="QMainWindow" name="homewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OVL-Configurator - Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_logo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>10</y>
      <width>260</width>
      <height>88</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>88</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>260</width>
      <height>88</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_email">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_login">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>80</y>
      <width>100</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Space Grotesk Light</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_pswd">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_send">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>230</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_account">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>260</y>
      <width>345</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Space Grotesk Light</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>If you don't already have an account on this instance :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_chgMode">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_tools">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>320</y>
      <width>345</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Space Grotesk Light</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>You can start the installation of all needed tools here :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_tools">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Tools</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
